prawns/governance/src/handlers/treasury.proposed.event.ts

```diff
@@ -33,7 +38,7 @@ export default (network: SubstrateNetwork) =>
 
     // proposer
     const account = await getOrCreateGovernanceAccount(ctx.store, {
-      id: getCallOriginAccount(ctx.event.call.origin, network),
+      id: publicKey,
```

Comment:
I *think* this should be `accountAddress`

Reply:
You're right 🤦